### PR TITLE
DOC-2654 Update docs to match updated UI for timed exam allowances

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-2654](https://openedx.atlassian.net/browse/DOC-2654)

https://openedx.atlassian.net/browse/SOL-1480 introduces changes to the UI for timed exam allowances. I updated shared/course_features/timed_exams.rst with updated screenshots and a minor adjustment to the procedure for granting learners more time - http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/course_features/timed_exams.html#grant-learners-more-time-for-a-timed-exam

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @douglashall 
- [ ] Subject matter expert: @jaakana
- [x] Doc team review (sanity check): @edx/doc

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by catong** [comment]
@grantgoodmanedX If you update screenshots, check in a new file with the same name to replace the old screenshot, so that it's updated everywhere it is used, rather than add a file with a new name.

**Comment by catong** [comment]
Other than the observation about the image files, looks good to me. 👍 once the graphics are updated.

File: en_us/shared/course_features/timed_exams.rst [hunk 1]
```diff
@@ -129,22 +129,21 @@ extra time to complete a timed exam.
...
 
-   .. image:: ../../../shared/images/inst_dash_special_exams.png
+   .. image:: ../../../shared/images/inst_dash_special_exams-0.png
     :alt: The Allowance Section in the Instructor Dashboard.
     :width: 600
 
 #. Select **Add Allowance**.
 
-   .. image:: ../../../shared/images/new_allowance.png
+   .. image:: ../../../shared/images/new_allowance-0.png
```
Comment (srpearce) [inline en_us/shared/course_features/timed_exams.rst]: This image seems pretty big to me—maybe shrink it to 250 or 300?

**Review by srpearce — APPROVED** [review]

**Comment by srpearce** [comment]
One comment about one of the screenshots (plus what @catong said); otherwise 👍

**Comment by grantgoodmanedX** [comment]
@douglashall I tagged you for review because you were involved in the related SOL-1480. Could you give this a quick sanity check review, please?

**Comment by douglashall** [comment]
👍

**Comment by grantgoodmanedX** [comment]
I merged it anyway, since the only error was the known python highlighting issue: /home/travis/build/edx/edx-documentation/en_us/install_operations/source/configuration/tpa/tpa_IDmap_api.rst:92: WARNING: Could not lex literal_block as "python". Highlighting skipped.

**Pull request merged**